### PR TITLE
Minor fixes to docs

### PR DESCRIPTION
**Pull request by ktavabi**
#### What does this implement/fix?
Minor fix to set the same expected `clipping` arg in `mne_browse_raw` help message as that of `raw.plot()` docstring. Along with additional _NB_ in `extract_label_time_course` as per @larsoner.

### CODE REVIEW

File: mne/source_estimate.py [hunk 1]
```diff
@@ -2686,6 +2686,11 @@ def extract_label_time_course(stcs, labels, src, mode='mean_flip',
         degrees from one stc to the next.
     ``'max'``
         Max value within each label.
+    
+    If encountering a ``'ValueError'`` due to mismatch between number of
```
Comment (drammock) [inline mne/source_estimate.py]:
```suggestion
    If encountering a ``ValueError`` due to mismatch between number of
```

File: mne/source_estimate.py [hunk 2]
```diff
@@ -2686,6 +2686,11 @@ def extract_label_time_course(stcs, labels, src, mode='mean_flip',
...
+    source points in the subject source space and computed ``'stc'`` object set
```
Comment (drammock) [inline mne/source_estimate.py]:
```suggestion
    source points in the subject source space and computed ``stc`` object set
```

File: mne/source_estimate.py [hunk 3]
```diff
@@ -2686,6 +2686,11 @@ def extract_label_time_course(stcs, labels, src, mode='mean_flip',
...
+    ``'src'`` argument to ``'fwd['src']'`` to ensure the source space is
```
Comment (drammock) [inline mne/source_estimate.py]:
```suggestion
    ``src`` argument to ``fwd['src']`` to ensure the source space is
```

File: mne/source_estimate.py (hunk 1 above)
Comment (drammock) [reply mne/source_estimate.py]: (`ValueError` is not a string, so let's not format it like a string)

**Comment by ktavabi** [comment]
Thanks, Dan! suggestions accepted.

File: mne/source_estimate.py [hunk 4]
```diff
@@ -2686,6 +2686,11 @@ def extract_label_time_course(stcs, labels, src, mode='mean_flip',
         degrees from one stc to the next.
     ``'max'``
         Max value within each label.
+
```
Comment (larsoner) [inline mne/source_estimate.py]: Blank link contains whitespace

https://travis-ci.org/mne-tools/mne-python/jobs/535526522#L3375

File: mne/source_estimate.py [hunk 5]
```diff
@@ -2686,6 +2686,10 @@ def extract_label_time_course(stcs, labels, src, mode='mean_flip',
         degrees from one stc to the next.
     ``'max'``
         Max value within each label.
+    If encountering a ``ValueError`` due to mismatch between number of
+    source points in the subject source space and computed ``stc`` object set
+    ``src`` argument to ``fwd['src']`` to ensure the source space is
+    compatible between forward and inverse routines.
```
Comment (massich) [inline mne/source_estimate.py]: The CI error seems legit, due to this warning in `source_estimate`:

```sh
/home/circleci/project/mne/source_estimate.py:docstring of mne.extract_label_time_course:67: WARNING: Definition list ends without a blank line; unexpected unindent.
```

Check your indentation or leed with an empty line.

**Comment by massich** [comment]
Otherwise, +1 to merge when Circle is green. 

Thx @ktavabi

**Comment by larsoner** [comment]
Thanks @ktavabi

**Pull request merged**